client/src/splashPage/HeroImage.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, Card, Tile } from '../styles/globalStyles';
+
+export default function Hero() {
+  return (
+    <HeroImage>
+      <ImageOverlay>
+        <h3>Band-Ayd</h3>
```

Comment:
@kalebal Can we change this to H1 so the headers follow the semantic hierarchy? But it can be whatever font size makes sense for this element. Also, there are some other headers that are being used where we could change the headers to different values.

Reply:
For sure! I was thinking h1 would be in the header/nav, but it makes since for this to be h1 since it's the most prominent on the page.